### ISSUE TITLE
License terms

### ISSUE THREAD

**Issue by wotupfoo** [knaufinator/6DOF-Rotary-Stewart-Motion-Simulator#21]
Hi,
I want to fork your project but I see in your controller code a copy of the RotaryStewartPlatorm code - like getAlpha(), setPos() without the license attribution anywhere I can find in your repo. 
Is this a derivative of that work, or did you write them and the other repo is a copy of yours (and there they didn't follow the license terms). 

Just trying to do the right thing as I move off of his code base to use some of yours instead since I like your SimHub|SimTool integration more. 

https://github.com/ThomasKNR/RotaryStewartPlatform/blob/master/src_arduino_code/platform.ino

**Comment by knaufinator** [comment]
yeah, do what ever ya want with the code my dude. If you do something cool, send a link my way to take a look at it.  have fun

**Comment by ChristianKnorr** [comment]
Why do you want to fork? And yes, please link it here 👍

**Comment by wotupfoo** [comment]
I'd rather not fork but my playing around has already deviated a lot from what's in the repo here.
It's always a debate about whether it should be a completely different code base in a branch or a fork.

Here it is so far, the only code difference is a bit of code cleanup to suite my liking.
One of the biggest differences is that I want to make the Stewart conversion a module that you can insert into a processing path or not. That way the driver board can be a direct simtool|simhub SMC3.ino type interface, or configured to be a similar system but with Stewart platform conversion as well.

The big "value add" right now is that I captured all my research into README.md markdown files.
For example, here is my research on Mixed Reality gaming (my goal is to have a Stewart platform with Mixed Reality).

https://github.com/wotupfoo/SimRacing/blob/main/webpages/Mixed%20Reality%20Cockpit/README_OpenXR.md
